src/server/routes/index.js

```diff
@@ -13,6 +13,10 @@ const jwtCheck = jwt({
 
 const router = express.Router();	// eslint-disable-line new-cap
 
+router.get('/', (req, res) => {
```

Comment:
Are you changing the URL of the health check? This will be important to standardize across microservices as our infrastructure becomes more complex.